scripts/migrateHistoricBids.ts

```diff
@@ -127,6 +127,7 @@ function mapBidtoBidCancelledMessage(
       account: bid.account, //account instead of signer
       bidNonce: bid.bidNonce,
       version: bid.version ?? "1.0", //set version 1.0 by default
+      cancelDate: bid.cancelDate ?? 0 
```

Comment:
Defaulting to `0` seems dangerous since there could be a "bid placed" message for this bid at time=100, but then there's now a cancel time 0.

Reply:
In this sense 0 is just a default value. It shows that the bid was cancelled, but no time could be attributed to it. I would prefer a default value of 0 over an undefined value, because it at least shows that the field was intentionally set this way (because the cancelation date has no provenance)

Reply:
When the aggregator consumes this, could this cause problems though? The data store is going to want to display a history of events, such as "Bid was placed on 4/20", "Bid was cancelled on 4/23"
If we pass a `0` on the cancel time for the bid, will it show "Bid was cancelled on 1/1/1973"

Seems like we are pushing more complexity onto the aggregator when we could define a better base case here when we have a bit more context.